src/components/Alert.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+export default function Alert({ children }) {
```

Comment:
This is very unusual. `{ children }` or what it really is `props.children` is used by react to include data to in your component. Usually it is used to render different JSX elements, have a look at this example: 
```javascript
function Sidebar({ children }) {

    return (
        <div className="sidebar">
            {children}        
         </div>
    )
}


<Sidebar>
    <nav>
      <a href="#">Home</a>        
      <a href="#">Posts</a>        
      <a href="#">Contact</a>    
    </nav>
</Sidebar>
```
I think a better name for your property might be `message` or `text`. Then you can make sure that it'll always be a string and the name would be more descriptive.
https://reactjs.org/docs/jsx-in-depth.html#children-in-jsx